raku-pod.html.markdown

```diff
@@ -5,9 +5,9 @@ contributors:
 filename: learnpod.pod6
```

Comment:
The [extension](https://github.com/perl6/problem-solving/blob/master/solutions/language/Path-to-Raku.md#extensions) for documentation is now `.rakudoc`, instead of `.pod6`, so this might be worth updating too.